IndomitableIbises/Bangazon-API-Sprint-1
# Filter Data to Show Inactive Users aka Users who don't have Orders

## Link to Ticket
Closes #11 

## Description of Proposed Changes

- Filter all customers by Inactive status which means they don't have orders
- Created an 'inactive_customers' view to show inactive customers
- Added url so we can see the inactive customers on the server

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout RC-Inactive
```
1.  Make sure your server isn't running
2.  In Bash/Terminal, run ./django_data.sh API faker_factory for **PC** or  django_data.sh API faker_factory for **Mac**
3.  Run server: python manage.py runserver
4.  Click on customer link and create a few customers -- DO NOT click active.  We want them to be inactive customers.  
5.  Return to root, and click on inactive_customers link.  You will see the inactive users you just created
6.  Review the code in views.py and url.py
7.  Ask me questions should you have any



## Impacted Areas in Application

List general components of the application that this PR will affect:

*views.py
*urls.py

## Mentions @username

@CashewRose @jcarter0149 @sirwin4 
## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
